src/main/java/ua/su/controllers/MedicalProcedureController.java

```diff
@@ -1,4 +1,47 @@
 package ua.su.controllers;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import ua.su.domain.MedicalProcedure;
+import ua.su.service.MedicalProcedureService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/", produces = MediaType.APPLICATION_JSON_VALUE)
 public class MedicalProcedureController {
-}
+
+    private final MedicalProcedureService medicalProcedureService;
+
+    public MedicalProcedureController(MedicalProcedureService medicalProcedureService) {
+        this.medicalProcedureService = medicalProcedureService;
+    }
+
+    @GetMapping(value = "/medical-procedures")
+    public List<MedicalProcedure> getMedicalProcedures() {
+        return medicalProcedureService.findAll();
+    }
+
+    @GetMapping(value = "/medical-procedures/{id}")
+    public MedicalProcedure getMedicalProcedure(@PathVariable Long id) {
+        return medicalProcedureService.getEntry(id);
+    }
+
+    @PostMapping(value = "/clinics/{clinicId}/medical-procedures")
```

Comment:
use underscore instead of dash in resource path : medical_procedures

Reply:
The dash has been switched by underscore.